xmppserver/src/main/java/org/jivesoftware/openfire/commands/admin/user/AddUser.java

```diff
@@ -72,34 +77,34 @@ public void execute(SessionData sessionData, Element command) {
         }
         catch (NullPointerException npe) {
```

Comment:
This catch block needs to account for an invalid JID. `newuser@invalid@domain` generates an error server-side that's returned as an unhandled internal-server-error to the client

Reply:
Fixed that problem in this command, and a couple of others.